x-pack/plugins/observability_solution/observability_ai_assistant/server/functions/elasticsearch.ts

```diff
@@ -46,7 +46,7 @@ export function registerElasticsearchFunction({
         body,
       });
 
-      return { content: response };
+      return { content: { response } };
```

Comment:
sometimes `transport.request` returns plaintext, so we need to wrap it in an object. otherwise we'll have an error somewhere else where we expect it to be an object.